test/gpu/fuse_mlir.cpp

```diff
@@ -770,6 +770,291 @@ TEST_CASE(conv_add_split_reduce_multi_use_conv)
     EXPECT(p1.sort() == p2.sort());
 }
 
+TEST_CASE(gemm_softmax_gemm)
```

Comment:
Please add a couple of lines of documentation, for p1 and p2.

Reply:
Done